### DISCUSSION TITLE
v1.4-experimental1

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/WindowsAppSDK#3653] [Announcements]
## 1.4 Experimental 1 (1.4.230518007-experimental1) 🧪

The 1.4 experimental release of Windows App SDK is now available! Releases in the experimental channel include features that are in the early stages of development. Experimental features may be removed from the next release, or may never be released. See [Windows App SDK release channels](https://docs.microsoft.com/windows/apps/windows-app-sdk/release-channels) for more information about the various release channels.

This release includes bug fixes from 1.3 and provides access to non-stable APIs and features for WinUI 3, System Backdrops, Content, Windowing, Input, and Controls.

**Check out the [release notes](https://learn.microsoft.com/windows/apps/windows-app-sdk/experimental-channel#version-14-experimental-140-experimental1) for all of the new and updated features, limitations, and known issues.**

### Updating to this version

The 1.4.230518007-experimental1 package is available at: https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.4.230518007-experimental1

Additionally, for those testing deployment of unpackaged apps, download the Windows App SDK Installers and MSIX packages [here](https://aka.ms/windowsappsdk/1.4/1.4.230518007-experimental1/Microsoft.WindowsAppRuntime.Redist.1.4.zip).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.4-exp1'>v1.4-experimental1</a>.</em>

**Comment by malstraem** [comment]
Short grid column and row definitions still breaks hot reload and not visible by autocomplete. When it will be fixed?

**Reply by duncanmacmichael** [reply]
Do you have a link to an open GitHub issue on this?

**Reply by malstraem** [reply]
@duncanmacmichael, [yes](https://github.com/microsoft/microsoft-ui-xaml/issues/5944)

**Reply by duncanmacmichael** [reply]
Great, thanks. @bpulliam are we able to give an update on this?
